BlueprintUI/Sources/Internal/ElementIdentifier.swift

```diff
@@ -41,55 +41,87 @@
 
  You will note that the identifiers remain stable, which ultimately ensures that views are reused.
  */
-struct ElementIdentifier: Hashable, CustomDebugStringConvertible {
+final class ElementIdentifier: Hashable, CustomDebugStringConvertible {
```

Comment:
Looks like the stack might be out of order or a need a rebase, as these changes are in #386.

Reply:
This is expected, we have a tree of PRs, not a single stack, so I had to pull this in to avoid more thrash back and forth in this PR then another one. We should be merging the ID one soon so this will go away

Reply:
Ah, got it.

Why not just stack them on top of each other?

Reply:
So Kareem and I were able to independently work on pulling in separate pieces last week